docs/Coiote_IoT_DM/Coiote_videos/How_to_simply_onboard_and_provision_IoT_devices.md

```diff
@@ -0,0 +1,19 @@
+# How to simply onboard and provision IoT devices?
+
+If you've struggled with getting your IoT devices securely connected to the cloud, you're not alone – it's a common challenge for many developers. But here's the good news: we've got a straightforward solution, and using the Coiote IoT Device Management Platform can make it even easier.
```

Comment:
> But here's the good news

`here's` suggest singular, but `good news` is plural

Reply:
Also, at the beginning you're talking about `struggling` and `challenges`, but then you're saying `we can make it even easier`. That `even` doesn't make any sense.